packages/table/src/Row.tsx

```diff
@@ -70,42 +79,46 @@ const modeStyles = {
 };
 
 const rowStyle = css`
+  border-color: var(--lg-table-row-border-color);
```

Comment:
This seems like a great place to use the `border` property since we're defining each value that the property would handle.